### ISSUE TITLE
CAD files for physical design missing

### ISSUE THREAD

**Issue by toonvdijck** [basicallysource/sorter#5]
Correct me if wrong but i can't find the CAD designs anywhere.
Is there a material list available with all the parts needed to build?

**Comment by toonvdijck** [comment]
@spencerhhubert

**Comment by spencerhhubert** [comment]
@toonvdijck You are not wrong, they're not in the repo. The mechanical aspects of the Nexus aren't good enough yet, I don't want anyone to waste their time building it for such a low level of reliability. I could privately share should you/someone be interested in further developing the CAD stuff and the open source it in the repo together, but I am no mechanical engineer.

**Comment by toonvdijck** [comment]
@spencerhhubert That would be lovely. I'm not a engineer but more a hobbyist with a lot of interest.
You can send me something by DM

**Comment by thehoff** [comment]
@spencerhhubert I would be keen to have a look at the CAD and have a go at this build completely understanding that it's a WIP. Reach out if you are cool to share the CAD :)

**Comment by effreek** [comment]
@spencerhhubert Any progress from above colabs or is this still needing refining? I'm an engineer and love what looks like a good implementation idea for the bins/sorting into acro-mil style bins. Any way to get the CAD stuff?

**Comment by spencerhhubert** [comment]
The CAD for v1 is too messy to reasonable be shared

Technically it's a [public document](https://cad.onshape.com/documents/57a6deba5df3f2fefb14bfa4/w/69c1555983f7ea624f0cf5a5/e/62f915a1f9533b22259df854) but I would not recommend trying to recreate
